aodntools/timeseries_products/velocity_hourly_timeseries_template.json

```diff
@@ -0,0 +1,218 @@
+            },
+
+        "WCUR": {
+            "coordinates": "TIME DEPTH LATITUDE LONGITUDE",
+            "ancillary_variables": "WCUR_max, WCUR_min, WCUR_std, WCUR_count",
+            "long_name": "upward_sea_water_velocity",
+            "standard_name": "upward_sea_water_velocity",
+            "units": "m s-1",
+            "valid_max": 5.0,
+            "valid_min": -5.0
+            },
+        "WCUR_max": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "max data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:max"
+            },
+        "WCUR_min": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "min data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:min"
+            },
+        "WCUR_std": {
+            "units": "m s-1",
+            "standard_name": "upward_sea_water_velocity",
+            "long_name": "std data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:std"
+            },
+        "WCUR_count": {
+            "standard_name": "upward_sea_water_velocity number_of_observations",
+            "units": "1",
+            "long_name": "count data value in the bin, after rejection of flagged data",
+            "cell_methods": "TIME:count"
+            },
+        "CELL_INDEX": {
+            "long_name": "index of the corresponding measuring cell",
+            "comment": "the closest cell to the sensor is cell 0"
+        },
+        "SECONDS_TO_MIDDLE": {
+            "long_name": "number of seconds from timestamp to middle of the measurement window"
+        }
+    },
+    "_global":{
+        "abstract": "Hourly Time-series Product: This file contains all measurements of quality controled U, V and W sea water velocity variables from all instruments deployed at the selected site. Timestamps are chronologically ordered, and binned into 1 hour time intervals. Instrument details are stored as a variable in order to keep a record of the origin of each measurement. Out-of-water measurements have been excluded.",
+        "acknowledgement": "Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: \"Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is a national collaborative research infrastructure, supported by the Australian Government.\" If relevant, also credit other organisations involved in collection of this particular datastream (as listed in 'credit' in the metadata record).",
+        "author": "Klein, Eduardo",
+        "author_email": "eduardo.kleinsalas@utas.edu.au",
+        "citation": "The citation in a list of references is: \"IMOS [year-of-data-download], [Title], [data-access-URL], accessed [date-of-access].\".",
+        "Conventions": "CF-1.6,IMOS-1.4",
```

Comment:
Maybe also add this as a comment:
```
        "comment": "Timestamps in the input files indicate the start of each sampling period (instrument-dependent; up to an hour in duration) and these have not been shifted to the centre of the period before binning. This could lead to an artificial shift of up to half an hour in the output data."
```